2024-07-08 02:15:16,380 ERROR 43438 nodejs.ClusterClientNoResponseError: client no response in 7218463ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 43438
hostname: stempestdeMac-mini.local

2024-07-08 04:17:47,181 ERROR 43438 nodejs.ClusterClientNoResponseError: client no response in 7210776ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 43438
hostname: stempestdeMac-mini.local

2024-07-08 06:19:08,496 ERROR 43438 nodejs.ClusterClientNoResponseError: client no response in 7221414ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 43438
hostname: stempestdeMac-mini.local

2024-07-08 08:20:25,177 ERROR 43438 nodejs.ClusterClientNoResponseError: client no response in 7216830ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 43438
hostname: stempestdeMac-mini.local

2024-07-08 09:01:34,172 ERROR 43438 nodejs.ClusterClientNoResponseError: client no response in 2408792ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 43438
hostname: stempestdeMac-mini.local

2024-08-23 13:20:31,491 ERROR 29158 nodejs.ClusterClientNoResponseError: client no response in 1813687ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29158
hostname: stempestdeMac-mini.local

2024-08-23 13:39:02,059 ERROR 29158 nodejs.ClusterClientNoResponseError: client no response in 1051001ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29158
hostname: stempestdeMac-mini.local

2024-08-23 14:01:41,057 ERROR 29158 nodejs.ClusterClientNoResponseError: client no response in 1298986ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29158
hostname: stempestdeMac-mini.local

2024-08-23 18:05:23,383 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 1810040ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-23 18:21:24,926 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 901823ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-23 19:05:13,863 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 2568993ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-23 21:06:15,784 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 7201945ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-23 21:36:14,560 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 1738954ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-23 23:38:19,376 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 7204954ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

2024-08-24 01:39:19,894 ERROR 29783 nodejs.ClusterClientNoResponseError: client no response in 7200979ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/xiaoshiliang/Downloads/redbookserver/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 29783
hostname: stempestdeMac-mini.local

